voluncheer/opportunityboard/fixtures/category_data.json

```diff
@@ -0,0 +1,309 @@
+[
+    {
+      "model":"opportunityboard.category",
+      "pk":1,
```

Comment:
I would expect the PK to be generated when data is added to the database, not something we should synthetically create. Can you explain your thinking around including it here?

Reply:
I just copied Doug's hardcoded categories and convert them into json file. I guess it is not necessary but it does make it more human readable.

Reply:
My point is those values may already exist in a database you are going to attempt to write them to. It isn't safe to assume those values are not already used.

Reply:
Removed.